examples/simple-test/main.py

```diff
@@ -0,0 +1,99 @@
+from pathlib import Path
+
+import fire
+import pyds
+from sparrow_datums import FrameAugmentedBoxes, PType, non_max_suppression
+
+import sparrow_mlpipes as mlp
+
+WIDTH = 1920
+HEIGHT = 1080
+FPS = 20
+
+
+def print_objects_pad(_, info, __):
+    # Intiallizing object counter with 0.
+    batch_meta = mlp.get_batch_meta(info)
+    pyds.nvds_acquire_meta_lock(batch_meta)
+    for frame in mlp.pyds_generator(batch_meta.frame_meta_list):
+        frame_meta = mlp.get_frame_meta(frame)
+        print(frame_meta.frame_num)
+        for user in mlp.pyds_generator(frame_meta.frame_user_meta_list):
+            user_meta = mlp.get_user_meta(user)
+            augmented_boxes = mlp.get_output_tensor(user_meta, 0)
+            boxes = non_max_suppression(
+                FrameAugmentedBoxes(
+                    augmented_boxes,
+                    ptype=PType.relative_tlbr,
+                    image_width=WIDTH,
+                    image_height=HEIGHT,
+                )
+            )
+            print(boxes.shape)
+            for box in boxes:
+                obj_meta = pyds.nvds_acquire_obj_meta_from_pool(batch_meta)
+                obj_meta.rect_params.left = WIDTH * box.x1
+                obj_meta.rect_params.top = HEIGHT * box.y1
+                obj_meta.rect_params.width = WIDTH * box.w
+                obj_meta.rect_params.height = HEIGHT * box.h
+                obj_meta.rect_params.border_width = 3
+                obj_meta.rect_params.border_color.set(0, box.label / 91, box.score, 0)
+                pyds.nvds_add_obj_meta_to_frame(frame_meta, obj_meta, None)
+
+    pyds.nvds_release_meta_lock(batch_meta)
+    return mlp.Gst.PadProbeReturn.OK
+
+
+def main(
```

Comment:
I wanted to test out the the "print" statement you suggested adding after this line (https://github.com/sparrowml/sparrow-mlpipes/blob/main/examples/onnx-nvinfer/main.py#L19) so I added in a few more bins until I was calling `print_objects_pad`. It might be overly complex (I could probably do without running inference for now). Still troubleshooting some.

Reply:
https://user-images.githubusercontent.com/63494615/176163609-cfe6395f-7f66-457c-82b3-28e4fddaaaa8.mp4